drivers/pinmux/Makefile

```diff
@@ -3,9 +3,11 @@ ccflags-y +=-I$(srctree)/drivers
 ccflags-$(CONFIG_PINMUX_QMSI) += -I$(CONFIG_QMSI_INSTALL_PATH)/include
 
 # Board initialization
+obj-$(CONFIG_PINMUX_QMSI) += pinmux_qmsi.o
```

Comment:
I was asked to move stm32 related configs close to each other. I noticed that a couple of pinmux lines were added to the end of the list and felt it was better to have them along with the other pinmux lines. Let me know if I have to revert it.